services/parser-database/connector.py

```diff
@@ -73,7 +102,7 @@ def save_keywords_in_memory(keywords, article):
         if keyword not in keywords_in_memory:
             keywords_in_memory[keyword] = []
         keywords_in_memory[keyword].append({
-            "articleNumber": article["articleNumber"],
+            "id": article["articleNumber"],
```

Comment:
Won't this cause a change in the response for either the `articles/byKeyword`? Was this a bug before or does all the callers of that endpoint need to change too?

Reply:
This was a bug that was fixed with another PR that was submitted.